src/test/java/com/stripe/model/AccountTest.java

```diff
@@ -38,26 +38,30 @@ public void testDeserialize() throws StripeException, IOException {
 		String json = resource("account.json");
 		Account acc = APIResource.GSON.fromJson(json, Account.class);
 
-		Account expected = new Account();
-		expected.id = "acct_1032D82eZvKYlo2C";
-		expected.chargesEnabled = false;
-		expected.detailsSubmitted = false;
-		expected.transfersEnabled = false;
-		expected.currenciesSupported = new LinkedList<String>();
-		expected.currenciesSupported.add("usd");
-		expected.currenciesSupported.add("aud");
-		expected.legalEntity = new LegalEntity();
-		expected.legalEntity.type = "company";
-		expected.legalEntity.address = new Address();
-		expected.legalEntity.address.city = "San Francisco";
-		expected.email = "site@stripe.com";
-		expected.defaultCurrency = "usd";
-		expected.country = "US";
-		expected.timezone = "US/Pacific";
-		expected.displayName = "Stripe.com";
-		expected.businessName = "Stripe.com";
-
-		assertEquals(expected, acc);
+        assertEquals("acct_1032D82eZvKYlo2C", acc.getId());
+        assertEquals(false, acc.getChargesEnabled());
+        assertEquals(false, acc.getDetailsSubmitted());
+        assertEquals(false, acc.getTransfersEnabled());
+
+		LinkedList<String> cs = new LinkedList<String>();
+		cs.add("usd");
+		cs.add("aud");
+        assertEquals(cs, acc.getCurrenciesSupported());
+
+        LegalEntity le = new LegalEntity();
```

Comment:
Since I changed enough of the file I'll convert it to 4 spaces. Still need to do the giant conversion off of tabs at some point.